<<<<<<< HEAD
CRITICAL - 2020-12-14 02:27:50 --> Use of undefined constant trues - assumed 'trues' (this will throw an Error in a future version of PHP)
#0 C:\xampp\htdocs\comple\app\Services\AlumnoLoginService.php(26): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Use of undefine...', 'C:\\xampp\\htdocs...', 26, Array)
#1 C:\xampp\htdocs\comple\app\Controllers\AlumnoLoginController.php(30): App\Services\AlumnoLoginService->login('161160163', '1111')
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\AlumnoLoginController->login()
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AlumnoLoginController))
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-12-14 02:54:39 --> Undefined variable: responsable
#0 C:\xampp\htdocs\comple\app\Services\ResponsableLoginService.php(37): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 37, Array)
#1 C:\xampp\htdocs\comple\app\Controllers\ResponsableLoginController.php(33): App\Services\ResponsableLoginService->login('RSMLUISAR4132', '12345678')
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\ResponsableLoginController->login()
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ResponsableLoginController))
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-12-14 02:55:54 --> Undefined variable: rfc_responsable_aux
#0 C:\xampp\htdocs\comple\app\Services\ResponsableLoginService.php(40): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 40, Array)
#1 C:\xampp\htdocs\comple\app\Controllers\ResponsableLoginController.php(33): App\Services\ResponsableLoginService->login('RSMLUISAR4132', '12345678')
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\ResponsableLoginController->login()
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ResponsableLoginController))
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-12-14 03:11:55 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\AlumnoModel.php(23): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Services\Responsable\AlumnoService.php(43): App\Models\Responsable\AlumnoModel->getResponsablePorRfc('RSMLUISAR4132')
#4 C:\xampp\htdocs\comple\app\Controllers\Responsable\AlumnoController.php(17): App\Services\Responsable\AlumnoService->getResponsablePorRfc('RSMLUISAR4132')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AlumnoController->index()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AlumnoController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-14 03:31:38 --> Call to undefined method App\Services\AlumnoLoginService::login()
#0 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\AlumnoLoginController->loginAlumno()
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AlumnoLoginController))
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2020-12-14 03:38:58 --> Undefined variable: alumno
#0 C:\xampp\htdocs\comple\app\Controllers\Alumno\InicioController.php(27): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 27, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Alumno\InicioController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno\InicioController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-14 03:40:28 --> Undefined variable: alumnos
#0 C:\xampp\htdocs\comple\app\Controllers\Alumno\InicioController.php(27): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 27, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Alumno\InicioController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno\InicioController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-14 03:40:30 --> Undefined variable: alumnos
#0 C:\xampp\htdocs\comple\app\Controllers\Alumno\InicioController.php(27): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 27, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Alumno\InicioController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno\InicioController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-14 03:40:56 --> Undefined variable: alumnos
#0 C:\xampp\htdocs\comple\app\Controllers\Alumno\InicioController.php(27): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 27, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Alumno\InicioController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno\InicioController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-14 03:44:01 --> Undefined variable: alumnos
#0 C:\xampp\htdocs\comple\app\Controllers\Alumno\InicioController.php(27): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 27, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Alumno\InicioController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno\InicioController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-14 03:45:53 --> Undefined variable: alumno
#0 C:\xampp\htdocs\comple\app\Controllers\Alumno\InicioController.php(27): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 27, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Alumno\InicioController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno\InicioController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-14 03:46:29 --> Undefined variable: alumnos
#0 C:\xampp\htdocs\comple\app\Controllers\Alumno\InicioController.php(27): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 27, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Alumno\InicioController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno\InicioController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-14 03:46:30 --> Undefined variable: alumnos
#0 C:\xampp\htdocs\comple\app\Controllers\Alumno\InicioController.php(27): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 27, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Alumno\InicioController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno\InicioController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-14 03:46:31 --> Undefined variable: alumnos
#0 C:\xampp\htdocs\comple\app\Controllers\Alumno\InicioController.php(27): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 27, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Alumno\InicioController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno\InicioController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-14 03:46:39 --> Too few arguments to function App\Services\Alumno\HistorialService::getActividades_no_calificadas(), 0 passed in C:\xampp\htdocs\comple\app\Controllers\Alumno\InicioController.php on line 27 and exactly 1 expected
#0 C:\xampp\htdocs\comple\app\Controllers\Alumno\InicioController.php(27): App\Services\Alumno\HistorialService->getActividades_no_calificadas()
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Alumno\InicioController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno\InicioController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-14 03:47:00 --> Undefined variable: alumnos
#0 C:\xampp\htdocs\comple\app\Services\Alumno\HistorialService.php(25): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 25, Array)
#1 C:\xampp\htdocs\comple\app\Controllers\Alumno\InicioController.php(27): App\Services\Alumno\HistorialService->getActividades_no_calificadas()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Alumno\InicioController->index()
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno\InicioController))
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-12-14 03:49:38 --> Undefined variable: alumno
#0 C:\xampp\htdocs\comple\app\Controllers\Alumno\InicioController.php(27): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 27, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Alumno\InicioController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno\InicioController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-14 03:49:40 --> Undefined variable: alumno
#0 C:\xampp\htdocs\comple\app\Controllers\Alumno\InicioController.php(27): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 27, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Alumno\InicioController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno\InicioController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-14 03:49:41 --> Undefined variable: alumno
#0 C:\xampp\htdocs\comple\app\Controllers\Alumno\InicioController.php(27): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 27, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Alumno\InicioController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno\InicioController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-14 03:49:42 --> Undefined variable: alumno
#0 C:\xampp\htdocs\comple\app\Controllers\Alumno\InicioController.php(27): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 27, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Alumno\InicioController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno\InicioController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-14 03:49:43 --> Undefined variable: alumno
#0 C:\xampp\htdocs\comple\app\Controllers\Alumno\InicioController.php(27): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 27, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Alumno\InicioController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno\InicioController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-14 03:49:44 --> Undefined variable: alumno
#0 C:\xampp\htdocs\comple\app\Controllers\Alumno\InicioController.php(27): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 27, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Alumno\InicioController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno\InicioController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-14 03:49:45 --> Undefined variable: alumno
#0 C:\xampp\htdocs\comple\app\Controllers\Alumno\InicioController.php(27): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 27, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Alumno\InicioController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno\InicioController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-14 03:57:44 --> Undefined property: stdClass::$loginresponsable
#0 C:\xampp\htdocs\comple\app\Controllers\Responsable\InicioController.php(17): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 17, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\InicioController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\InicioController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-14 03:58:00 --> Undefined property: stdClass::$login
#0 C:\xampp\htdocs\comple\app\Controllers\Responsable\InicioController.php(17): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 17, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\InicioController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\InicioController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-14 03:58:13 --> Undefined property: stdClass::$login
#0 C:\xampp\htdocs\comple\app\Controllers\Responsable\InicioController.php(17): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 17, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\InicioController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\InicioController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-14 03:59:13 --> Undefined variable: rfc_responsable
#0 C:\xampp\htdocs\comple\app\Controllers\Responsable\InicioController.php(19): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 19, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\InicioController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\InicioController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-14 04:02:55 --> Undefined variable: rfc_responsable
#0 C:\xampp\htdocs\comple\app\Controllers\Responsable\InicioController.php(19): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 19, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\InicioController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\InicioController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-14 04:02:56 --> Undefined variable: rfc_responsable
#0 C:\xampp\htdocs\comple\app\Controllers\Responsable\InicioController.php(19): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 19, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\InicioController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\InicioController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-14 04:03:09 --> Undefined variable: rfc_responsable
#0 C:\xampp\htdocs\comple\app\Controllers\Responsable\InicioController.php(19): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 19, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\InicioController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\InicioController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-14 04:03:44 --> Undefined variable: rfc_responsable
#0 C:\xampp\htdocs\comple\app\Controllers\Responsable\InicioController.php(19): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 19, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\InicioController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\InicioController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-14 04:03:57 --> Undefined variable: rfc_responsable
#0 C:\xampp\htdocs\comple\app\Controllers\Responsable\InicioController.php(19): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 19, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\InicioController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\InicioController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-14 04:04:49 --> Undefined variable: rfc_responsable
#0 C:\xampp\htdocs\comple\app\Controllers\Responsable\InicioController.php(19): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 19, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\InicioController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\InicioController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-14 04:05:49 --> Undefined variable: rfc_responsable
#0 C:\xampp\htdocs\comple\app\Controllers\Responsable\InicioController.php(19): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 19, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\InicioController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\InicioController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-14 04:07:41 --> Too few arguments to function App\Services\Responsable\InicioService::getActividadesPorResponsable(), 0 passed in C:\xampp\htdocs\comple\app\Controllers\Responsable\InicioController.php on line 19 and exactly 1 expected
#0 C:\xampp\htdocs\comple\app\Controllers\Responsable\InicioController.php(19): App\Services\Responsable\InicioService->getActividadesPorResponsable()
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\InicioController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\InicioController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-14 04:16:45 --> Undefined variable: alumnos
#0 C:\xampp\htdocs\comple\app\Controllers\Alumno\InicioController.php(27): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 27, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Alumno\InicioController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno\InicioController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-14 04:17:43 --> Undefined variable: alumnos
#0 C:\xampp\htdocs\comple\app\Controllers\Alumno\InicioController.php(27): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 27, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Alumno\InicioController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno\InicioController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-14 04:17:59 --> Too few arguments to function App\Services\Alumno\HistorialService::getActividades_no_calificadas(), 0 passed in C:\xampp\htdocs\comple\app\Controllers\Alumno\InicioController.php on line 27 and exactly 1 expected
#0 C:\xampp\htdocs\comple\app\Controllers\Alumno\InicioController.php(27): App\Services\Alumno\HistorialService->getActividades_no_calificadas()
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Alumno\InicioController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno\InicioController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-14 04:19:20 --> Undefined variable: num_control
#0 C:\xampp\htdocs\comple\app\Controllers\Alumno\InscripcionController.php(20): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 20, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Alumno\InscripcionController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno\InscripcionController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-14 20:02:02 --> Maximum execution time of 120 seconds exceeded
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-12-14 20:03:35 --> MySQL server has gone away
#0 C:\xampp\htdocs\comple\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT `num_con...')
#1 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `num_con...')
#2 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `num_con...')
#3 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `num_con...', Array, false)
#4 C:\xampp\htdocs\comple\app\Models\AlumnoModel.php(21): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\comple\app\Services\Division\InscripcionService.php(193): App\Models\AlumnoModel->getAlumnoPorNoControl('201160077')
#6 C:\xampp\htdocs\comple\app\Controllers\Division\InscripcionController.php(26): App\Services\Division\InscripcionService->unirRegistros(Array)
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Division\InscripcionController->index()
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Division\InscripcionController))
#9 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2020-12-14 20:03:35 --> Undefined property: stdClass::$num_control
#0 C:\xampp\htdocs\comple\app\Controllers\Alumno\HistorialController.php(24): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 24, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Alumno\HistorialController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno\HistorialController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-14 20:12:27 --> Trying to get property 'num_control' of non-object
#0 C:\xampp\htdocs\comple\app\Controllers\Alumno\InicioController.php(24): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'C:\\xampp\\htdocs...', 24, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Alumno\InicioController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno\InicioController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-14 21:12:27 --> Maximum execution time of 120 seconds exceeded
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-12-14 23:05:39 --> Maximum execution time of 120 seconds exceeded
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-12-14 23:09:54 --> Undefined property: stdClass::$num_control
#0 C:\xampp\htdocs\comple\app\Controllers\Alumno\HistorialController.php(24): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 24, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Alumno\HistorialController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno\HistorialController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-14 23:09:54 --> Undefined property: stdClass::$num_control
#0 C:\xampp\htdocs\comple\app\Controllers\Alumno\InscripcionController.php(18): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 18, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Alumno\InscripcionController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno\InscripcionController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-14 23:09:54 --> Undefined property: stdClass::$num_control
#0 C:\xampp\htdocs\comple\app\Controllers\Alumno\HistorialController.php(24): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 24, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Alumno\HistorialController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno\HistorialController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-14 23:09:54 --> Undefined property: stdClass::$num_control
#0 C:\xampp\htdocs\comple\app\Controllers\Alumno\InicioController.php(24): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 24, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Alumno\InicioController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Alumno\InicioController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
=======
CRITICAL - 2020-12-14 01:28:02 --> Undefined property: stdClass::$nombre_responsable
#0 C:\xampp\htdocs\comple\app\Views\Jefes\Constancia\listar.php(76): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 76, Array)
#1 C:\xampp\htdocs\comple\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\comple\system\Common.php(1119): CodeIgniter\View\View->render('jefes/constanci...', Array, true)
#3 C:\xampp\htdocs\comple\app\Controllers\Jefes\ConstanciaController.php(41): view('jefes/constanci...', Array)
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Jefes\ConstanciaController->constancia()
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Jefes\ConstanciaController))
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-12-14 01:28:19 --> Undefined variable: ac
#0 C:\xampp\htdocs\comple\app\Views\Jefes\Constancia\listar.php(76): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 76, Array)
#1 C:\xampp\htdocs\comple\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\comple\system\Common.php(1119): CodeIgniter\View\View->render('jefes/constanci...', Array, true)
#3 C:\xampp\htdocs\comple\app\Controllers\Jefes\ConstanciaController.php(41): view('jefes/constanci...', Array)
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Jefes\ConstanciaController->constancia()
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Jefes\ConstanciaController))
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-12-14 02:55:22 --> Undefined property: stdClass::$estatus
#0 C:\xampp\htdocs\comple\app\Views\Jefes\Responsables\listar.php(54): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 54, Array)
#1 C:\xampp\htdocs\comple\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\comple\system\Common.php(1119): CodeIgniter\View\View->render('Jefes/Responsab...', Array, true)
#3 C:\xampp\htdocs\comple\app\Controllers\Jefes\ResponsableController.php(22): view('Jefes/Responsab...', Array)
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Jefes\ResponsableController->index()
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Jefes\ResponsableController))
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-12-14 03:17:19 --> Unknown column 'id_area' in 'where clause'
#0 C:\xampp\htdocs\comple\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT *\nFROM `...')
#1 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\comple\app\Models\Jefes\ResponsableModel.php(19): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\comple\app\Services\Jefes\ResponsableService.php(21): App\Models\Jefes\ResponsableModel->getResponsables('2')
#6 C:\xampp\htdocs\comple\app\Controllers\Jefes\ResponsableController.php(18): App\Services\Jefes\ResponsableService->getResponsables('2')
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Jefes\ResponsableController->index()
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Jefes\ResponsableController))
#9 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2020-12-14 03:19:37 --> Trying to get property 'estatus' of non-object
#0 C:\xampp\htdocs\comple\app\Services\Jefes\ResponsableService.php(82): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'C:\\xampp\\htdocs...', 82, Array)
#1 C:\xampp\htdocs\comple\app\Controllers\Jefes\ResponsableController.php(166): App\Services\Jefes\ResponsableService->cambiarEstatus('')
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Jefes\ResponsableController->cambiarEstatus()
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Jefes\ResponsableController))
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-12-14 03:23:04 --> Trying to get property 'estatus' of non-object
#0 C:\xampp\htdocs\comple\app\Services\Jefes\ResponsableService.php(82): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'C:\\xampp\\htdocs...', 82, Array)
#1 C:\xampp\htdocs\comple\app\Controllers\Jefes\ResponsableController.php(166): App\Services\Jefes\ResponsableService->cambiarEstatus('')
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Jefes\ResponsableController->cambiarEstatus()
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Jefes\ResponsableController))
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-12-14 03:27:00 --> Undefined property: stdClass::$rfc_responsable
#0 C:\xampp\htdocs\comple\app\Views\Jefes\Responsables\listar.php(48): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 48, Array)
#1 C:\xampp\htdocs\comple\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\comple\system\Common.php(1119): CodeIgniter\View\View->render('Jefes/Responsab...', Array, true)
#3 C:\xampp\htdocs\comple\app\Controllers\Jefes\ResponsableController.php(22): view('Jefes/Responsab...', Array)
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Jefes\ResponsableController->index()
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Jefes\ResponsableController))
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-12-14 03:28:37 --> Unknown column 'rfc_responsable' in 'where clause'
#0 C:\xampp\htdocs\comple\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT *\nFROM `...')
#1 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\comple\app\Models\Jefes\ResponsableModel.php(42): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\comple\app\Services\Jefes\ResponsableService.php(80): App\Models\Jefes\ResponsableModel->getResponsablePorRfc('RSMLUISAR4132')
#6 C:\xampp\htdocs\comple\app\Controllers\Jefes\ResponsableController.php(166): App\Services\Jefes\ResponsableService->cambiarEstatus('RSMLUISAR4132')
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Jefes\ResponsableController->cambiarEstatus()
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Jefes\ResponsableController))
#9 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2020-12-14 03:31:30 --> Unknown column 'id_area' in 'where clause'
#0 C:\xampp\htdocs\comple\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT *\nFROM `...')
#1 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\comple\app\Models\Jefes\ResponsableModel.php(16): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\comple\app\Services\Jefes\ResponsableService.php(21): App\Models\Jefes\ResponsableModel->getResponsables('2')
#6 C:\xampp\htdocs\comple\app\Controllers\Jefes\ResponsableController.php(18): App\Services\Jefes\ResponsableService->getResponsables('2')
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Jefes\ResponsableController->index()
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Jefes\ResponsableController))
#9 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2020-12-14 04:13:03 --> Unknown column 'r.id_area' in 'on clause'
#0 C:\xampp\htdocs\comple\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT `r`.`rfc...')
#1 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `r`.`rfc...')
#2 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `r`.`rfc...')
#3 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `r`.`rfc...', Array, false)
#4 C:\xampp\htdocs\comple\app\Models\Jefes\ResponsableModel.php(21): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\comple\app\Services\Jefes\ResponsableService.php(21): App\Models\Jefes\ResponsableModel->getResponsables('2')
#6 C:\xampp\htdocs\comple\app\Controllers\Jefes\ResponsableController.php(18): App\Services\Jefes\ResponsableService->getResponsables('2')
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Jefes\ResponsableController->index()
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Jefes\ResponsableController))
#9 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2020-12-14 04:13:23 --> Unknown column 'r.id_area' in 'where clause'
#0 C:\xampp\htdocs\comple\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT `r`.`rfc...')
#1 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `r`.`rfc...')
#2 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `r`.`rfc...')
#3 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `r`.`rfc...', Array, false)
#4 C:\xampp\htdocs\comple\app\Models\Jefes\ResponsableModel.php(21): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\comple\app\Services\Jefes\ResponsableService.php(21): App\Models\Jefes\ResponsableModel->getResponsables('2')
#6 C:\xampp\htdocs\comple\app\Controllers\Jefes\ResponsableController.php(18): App\Services\Jefes\ResponsableService->getResponsables('2')
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Jefes\ResponsableController->index()
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Jefes\ResponsableController))
#9 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2020-12-14 04:13:52 --> Unknown column 'r.id_area' in 'where clause'
#0 C:\xampp\htdocs\comple\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT `r`.`rfc...')
#1 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `r`.`rfc...')
#2 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `r`.`rfc...')
#3 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `r`.`rfc...', Array, false)
#4 C:\xampp\htdocs\comple\app\Models\Jefes\ResponsableModel.php(21): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\comple\app\Services\Jefes\ResponsableService.php(21): App\Models\Jefes\ResponsableModel->getResponsables('2')
#6 C:\xampp\htdocs\comple\app\Controllers\Jefes\ResponsableController.php(18): App\Services\Jefes\ResponsableService->getResponsables('2')
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Jefes\ResponsableController->index()
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Jefes\ResponsableController))
#9 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2020-12-14 04:14:03 --> Undefined property: stdClass::$estatus
#0 C:\xampp\htdocs\comple\app\Views\Jefes\Responsables\listar.php(54): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 54, Array)
#1 C:\xampp\htdocs\comple\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\comple\system\Common.php(1119): CodeIgniter\View\View->render('Jefes/Responsab...', Array, true)
#3 C:\xampp\htdocs\comple\app\Controllers\Jefes\ResponsableController.php(22): view('Jefes/Responsab...', Array)
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Jefes\ResponsableController->index()
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Jefes\ResponsableController))
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-12-14 04:15:34 --> Unknown column 'r.id_area' in 'where clause'
#0 C:\xampp\htdocs\comple\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT `r`.`rfc...')
#1 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `r`.`rfc...')
#2 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `r`.`rfc...')
#3 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `r`.`rfc...', Array, false)
#4 C:\xampp\htdocs\comple\app\Models\Jefes\ResponsableModel.php(21): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\comple\app\Services\Jefes\ResponsableService.php(21): App\Models\Jefes\ResponsableModel->getResponsables('2')
#6 C:\xampp\htdocs\comple\app\Controllers\Jefes\ResponsableController.php(18): App\Services\Jefes\ResponsableService->getResponsables('2')
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Jefes\ResponsableController->index()
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Jefes\ResponsableController))
#9 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2020-12-14 04:15:47 --> Unknown column 'r.id_actividad' in 'on clause'
#0 C:\xampp\htdocs\comple\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT `r`.`rfc...')
#1 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `r`.`rfc...')
#2 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `r`.`rfc...')
#3 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `r`.`rfc...', Array, false)
#4 C:\xampp\htdocs\comple\app\Models\Jefes\ResponsableModel.php(21): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\comple\app\Services\Jefes\ResponsableService.php(21): App\Models\Jefes\ResponsableModel->getResponsables('2')
#6 C:\xampp\htdocs\comple\app\Controllers\Jefes\ResponsableController.php(18): App\Services\Jefes\ResponsableService->getResponsables('2')
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Jefes\ResponsableController->index()
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Jefes\ResponsableController))
#9 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2020-12-14 04:18:16 --> Unknown column 'r.id_area' in 'on clause'
#0 C:\xampp\htdocs\comple\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT `a`.`id_...')
#1 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `a`.`id_...')
#2 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `a`.`id_...')
#3 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `a`.`id_...', Array, false)
#4 C:\xampp\htdocs\comple\app\Models\Jefes\ResponsableModel.php(21): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\comple\app\Services\Jefes\ResponsableService.php(21): App\Models\Jefes\ResponsableModel->getResponsables('2')
#6 C:\xampp\htdocs\comple\app\Controllers\Jefes\ResponsableController.php(18): App\Services\Jefes\ResponsableService->getResponsables('2')
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Jefes\ResponsableController->index()
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Jefes\ResponsableController))
#9 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2020-12-14 04:34:15 --> Undefined property: stdClass::$nombre
#0 C:\xampp\htdocs\comple\app\Views\Jefes\Responsables\listar.php(49): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 49, Array)
#1 C:\xampp\htdocs\comple\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\comple\system\Common.php(1119): CodeIgniter\View\View->render('Jefes/Responsab...', Array, true)
#3 C:\xampp\htdocs\comple\app\Controllers\Jefes\ResponsableController.php(22): view('Jefes/Responsab...', Array)
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Jefes\ResponsableController->index()
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Jefes\ResponsableController))
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-12-14 19:22:21 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `u`.`usu...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\UsuarioModel.php(21): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Services\UsuarioService.php(22): App\Models\UsuarioModel->getUsuarioPorUsuario('ESCOLARES')
#4 C:\xampp\htdocs\comple\app\Controllers\UsuarioController.php(33): App\Services\UsuarioService->login('ESCOLARES', '12345678')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\UsuarioController->login()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuarioController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-14 19:26:18 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `u`.`usu...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\UsuarioModel.php(21): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Services\UsuarioService.php(22): App\Models\UsuarioModel->getUsuarioPorUsuario('ESCOLARES')
#4 C:\xampp\htdocs\comple\app\Controllers\UsuarioController.php(33): App\Services\UsuarioService->login('ESCOLARES', '12345678')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\UsuarioController->login()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuarioController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-14 19:28:39 --> Maximum execution time of 120 seconds exceeded
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-12-14 19:29:46 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `u`.`usu...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\UsuarioModel.php(21): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Services\UsuarioService.php(22): App\Models\UsuarioModel->getUsuarioPorUsuario('ESCOLARES')
#4 C:\xampp\htdocs\comple\app\Controllers\UsuarioController.php(33): App\Services\UsuarioService->login('ESCOLARES', '12345678')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\UsuarioController->login()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuarioController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-14 19:32:35 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `u`.`usu...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\UsuarioModel.php(21): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Services\UsuarioService.php(22): App\Models\UsuarioModel->getUsuarioPorUsuario('ESCOLARES')
#4 C:\xampp\htdocs\comple\app\Controllers\UsuarioController.php(33): App\Services\UsuarioService->login('ESCOLARES', '12345678')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\UsuarioController->login()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuarioController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-14 20:34:35 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `u`.`usu...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\UsuarioModel.php(21): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Services\UsuarioService.php(22): App\Models\UsuarioModel->getUsuarioPorUsuario('ESCOLARES')
#4 C:\xampp\htdocs\comple\app\Controllers\UsuarioController.php(33): App\Services\UsuarioService->login('ESCOLARES', '12345678')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\UsuarioController->login()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuarioController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-14 20:36:28 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Escolares\GenerarModel.php(19): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Services\Escolares\GenerarService.php(16): App\Models\Escolares\GenerarModel->getAlumnoPorNoControl('161160176')
#4 C:\xampp\htdocs\comple\app\Controllers\Escolares\GenerarController.php(25): App\Services\Escolares\GenerarService->getAlumnoPorNoControl('161160176')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\GenerarController->constanciaParcial()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\GenerarController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-14 20:39:06 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Escolares\GenerarModel.php(19): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Services\Escolares\GenerarService.php(16): App\Models\Escolares\GenerarModel->getAlumnoPorNoControl('161160176')
#4 C:\xampp\htdocs\comple\app\Controllers\Escolares\GenerarController.php(25): App\Services\Escolares\GenerarService->getAlumnoPorNoControl('161160176')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\GenerarController->constanciaParcial()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\GenerarController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-14 20:39:10 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Escolares\GenerarModel.php(19): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Services\Escolares\GenerarService.php(16): App\Models\Escolares\GenerarModel->getAlumnoPorNoControl('161160176')
#4 C:\xampp\htdocs\comple\app\Controllers\Escolares\GenerarController.php(25): App\Services\Escolares\GenerarService->getAlumnoPorNoControl('161160176')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\GenerarController->constanciaParcial()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\GenerarController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-14 20:39:26 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Escolares\GenerarModel.php(19): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Services\Escolares\GenerarService.php(16): App\Models\Escolares\GenerarModel->getAlumnoPorNoControl('161160176')
#4 C:\xampp\htdocs\comple\app\Controllers\Escolares\GenerarController.php(25): App\Services\Escolares\GenerarService->getAlumnoPorNoControl('161160176')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\GenerarController->constanciaParcial()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\GenerarController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-14 20:39:32 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Escolares\GenerarModel.php(19): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Services\Escolares\GenerarService.php(16): App\Models\Escolares\GenerarModel->getAlumnoPorNoControl('161160176')
#4 C:\xampp\htdocs\comple\app\Controllers\Escolares\GenerarController.php(25): App\Services\Escolares\GenerarService->getAlumnoPorNoControl('161160176')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\GenerarController->constanciaParcial()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\GenerarController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-14 20:39:47 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Escolares\ActividadModel.php(19): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Services\Escolares\ActividadService.php(16): App\Models\Escolares\ActividadModel->getAlumnoPorNoControl('161160176')
#4 C:\xampp\htdocs\comple\app\Controllers\Escolares\ActividadController.php(29): App\Services\Escolares\ActividadService->getAlumnoPorNoControl('161160176')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\ActividadController->buscar()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\ActividadController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-14 20:40:28 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Escolares\ActividadModel.php(19): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Services\Escolares\ActividadService.php(16): App\Models\Escolares\ActividadModel->getAlumnoPorNoControl('161160176')
#4 C:\xampp\htdocs\comple\app\Controllers\Escolares\ActividadController.php(29): App\Services\Escolares\ActividadService->getAlumnoPorNoControl('161160176')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\ActividadController->buscar()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\ActividadController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-14 20:40:30 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Escolares\ActividadModel.php(19): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Services\Escolares\ActividadService.php(16): App\Models\Escolares\ActividadModel->getAlumnoPorNoControl('161160176')
#4 C:\xampp\htdocs\comple\app\Controllers\Escolares\ActividadController.php(29): App\Services\Escolares\ActividadService->getAlumnoPorNoControl('161160176')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\ActividadController->buscar()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\ActividadController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-14 20:40:31 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Escolares\ActividadModel.php(19): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Services\Escolares\ActividadService.php(16): App\Models\Escolares\ActividadModel->getAlumnoPorNoControl('161160176')
#4 C:\xampp\htdocs\comple\app\Controllers\Escolares\ActividadController.php(29): App\Services\Escolares\ActividadService->getAlumnoPorNoControl('161160176')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\ActividadController->buscar()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\ActividadController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-14 20:40:32 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Escolares\ActividadModel.php(19): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Services\Escolares\ActividadService.php(16): App\Models\Escolares\ActividadModel->getAlumnoPorNoControl('161160176')
#4 C:\xampp\htdocs\comple\app\Controllers\Escolares\ActividadController.php(29): App\Services\Escolares\ActividadService->getAlumnoPorNoControl('161160176')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\ActividadController->buscar()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\ActividadController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-14 20:40:59 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Escolares\ActividadModel.php(19): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Services\Escolares\ActividadService.php(16): App\Models\Escolares\ActividadModel->getAlumnoPorNoControl('161160176')
#4 C:\xampp\htdocs\comple\app\Controllers\Escolares\ActividadController.php(29): App\Services\Escolares\ActividadService->getAlumnoPorNoControl('161160176')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\ActividadController->buscar()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\ActividadController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-14 20:41:19 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Escolares\ActividadModel.php(19): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Services\Escolares\ActividadService.php(16): App\Models\Escolares\ActividadModel->getAlumnoPorNoControl('161160176')
#4 C:\xampp\htdocs\comple\app\Controllers\Escolares\ActividadController.php(29): App\Services\Escolares\ActividadService->getAlumnoPorNoControl('161160176')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\ActividadController->buscar()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\ActividadController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-14 20:41:29 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Escolares\ActividadModel.php(19): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Services\Escolares\ActividadService.php(16): App\Models\Escolares\ActividadModel->getAlumnoPorNoControl('161160176')
#4 C:\xampp\htdocs\comple\app\Controllers\Escolares\ActividadController.php(29): App\Services\Escolares\ActividadService->getAlumnoPorNoControl('161160176')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\ActividadController->buscar()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\ActividadController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-14 20:41:39 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Escolares\ActividadModel.php(19): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Services\Escolares\ActividadService.php(16): App\Models\Escolares\ActividadModel->getAlumnoPorNoControl('161160176')
#4 C:\xampp\htdocs\comple\app\Controllers\Escolares\ActividadController.php(29): App\Services\Escolares\ActividadService->getAlumnoPorNoControl('161160176')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\ActividadController->buscar()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\ActividadController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-14 20:41:49 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Escolares\ActividadModel.php(19): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Services\Escolares\ActividadService.php(16): App\Models\Escolares\ActividadModel->getAlumnoPorNoControl('161160176')
#4 C:\xampp\htdocs\comple\app\Controllers\Escolares\ActividadController.php(29): App\Services\Escolares\ActividadService->getAlumnoPorNoControl('161160176')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\ActividadController->buscar()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\ActividadController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-14 20:41:59 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Escolares\ActividadModel.php(19): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Services\Escolares\ActividadService.php(16): App\Models\Escolares\ActividadModel->getAlumnoPorNoControl('161160176')
#4 C:\xampp\htdocs\comple\app\Controllers\Escolares\ActividadController.php(29): App\Services\Escolares\ActividadService->getAlumnoPorNoControl('161160176')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\ActividadController->buscar()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\ActividadController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-14 20:42:09 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Escolares\ActividadModel.php(19): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Services\Escolares\ActividadService.php(16): App\Models\Escolares\ActividadModel->getAlumnoPorNoControl('161160176')
#4 C:\xampp\htdocs\comple\app\Controllers\Escolares\ActividadController.php(29): App\Services\Escolares\ActividadService->getAlumnoPorNoControl('161160176')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\ActividadController->buscar()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\ActividadController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-14 20:43:10 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Escolares\ActividadModel.php(19): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Services\Escolares\ActividadService.php(16): App\Models\Escolares\ActividadModel->getAlumnoPorNoControl('161160176')
#4 C:\xampp\htdocs\comple\app\Controllers\Escolares\ActividadController.php(29): App\Services\Escolares\ActividadService->getAlumnoPorNoControl('161160176')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\ActividadController->buscar()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\ActividadController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-14 20:43:14 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Escolares\ActividadModel.php(19): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Services\Escolares\ActividadService.php(16): App\Models\Escolares\ActividadModel->getAlumnoPorNoControl('161160176')
#4 C:\xampp\htdocs\comple\app\Controllers\Escolares\ActividadController.php(29): App\Services\Escolares\ActividadService->getAlumnoPorNoControl('161160176')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\ActividadController->buscar()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\ActividadController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-14 20:43:15 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Escolares\ActividadModel.php(19): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Services\Escolares\ActividadService.php(16): App\Models\Escolares\ActividadModel->getAlumnoPorNoControl('161160176')
#4 C:\xampp\htdocs\comple\app\Controllers\Escolares\ActividadController.php(29): App\Services\Escolares\ActividadService->getAlumnoPorNoControl('161160176')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\ActividadController->buscar()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\ActividadController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-14 20:43:16 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Escolares\ActividadModel.php(19): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Services\Escolares\ActividadService.php(16): App\Models\Escolares\ActividadModel->getAlumnoPorNoControl('161160176')
#4 C:\xampp\htdocs\comple\app\Controllers\Escolares\ActividadController.php(29): App\Services\Escolares\ActividadService->getAlumnoPorNoControl('161160176')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\ActividadController->buscar()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\ActividadController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-14 21:09:37 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `a`.`id_...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Jefes\ConstanciaModel.php(36): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Services\Jefes\ConstanciaService.php(35): App\Models\Jefes\ConstanciaModel->getActividad('201160074', '12')
#4 C:\xampp\htdocs\comple\app\Controllers\Jefes\ConstanciaController.php(25): App\Services\Jefes\ConstanciaService->getActividad('201160074', '12')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Jefes\ConstanciaController->constancia()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Jefes\ConstanciaController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-14 23:00:06 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `a`.`id_...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Jefes\ConstanciaModel.php(36): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Services\Jefes\ConstanciaService.php(35): App\Models\Jefes\ConstanciaModel->getActividad('161160176', '7')
#4 C:\xampp\htdocs\comple\app\Controllers\Jefes\ConstanciaController.php(25): App\Services\Jefes\ConstanciaService->getActividad('161160176', '7')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Jefes\ConstanciaController->constancia()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Jefes\ConstanciaController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-14 23:00:12 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `a`.`id_...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Jefes\ConstanciaModel.php(36): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Services\Jefes\ConstanciaService.php(35): App\Models\Jefes\ConstanciaModel->getActividad('161160176', '7')
#4 C:\xampp\htdocs\comple\app\Controllers\Jefes\ConstanciaController.php(25): App\Services\Jefes\ConstanciaService->getActividad('161160176', '7')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Jefes\ConstanciaController->constancia()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Jefes\ConstanciaController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-14 23:00:25 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `a`.`id_...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Jefes\ConstanciaModel.php(36): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Services\Jefes\ConstanciaService.php(35): App\Models\Jefes\ConstanciaModel->getActividad('161160176', '7')
#4 C:\xampp\htdocs\comple\app\Controllers\Jefes\ConstanciaController.php(25): App\Services\Jefes\ConstanciaService->getActividad('161160176', '7')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Jefes\ConstanciaController->constancia()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Jefes\ConstanciaController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-14 23:05:38 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Jefes\AlumnoModel.php(20): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Jefes\AlumnoModel.php(51): App\Models\Jefes\AlumnoModel->get_inscripciones('7')
#4 C:\xampp\htdocs\comple\app\Controllers\Jefes\AlumnoController.php(16): App\Models\Jefes\AlumnoModel->get_actividad_alumno('7')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Jefes\AlumnoController->index()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Jefes\AlumnoController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-14 23:57:17 --> file_get_contents(public/img/3logo.png): failed to open stream: No such file or directory
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'file_get_conten...', 'C:\\xampp\\htdocs...', 99, Array)
#1 C:\xampp\htdocs\comple\app\Views\Jefes\Constancia\listar.php(99): file_get_contents('public/img/3log...')
#2 C:\xampp\htdocs\comple\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\comple\system\Common.php(1119): CodeIgniter\View\View->render('jefes/constanci...', Array, true)
#4 C:\xampp\htdocs\comple\app\Controllers\Jefes\ConstanciaController.php(43): view('jefes/constanci...', Array)
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Jefes\ConstanciaController->constancia()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Jefes\ConstanciaController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-14 23:58:26 --> file_get_contents(public/img/3logos..jpeg): failed to open stream: No such file or directory
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'file_get_conten...', 'C:\\xampp\\htdocs...', 99, Array)
#1 C:\xampp\htdocs\comple\app\Views\Jefes\Constancia\listar.php(99): file_get_contents('public/img/3log...')
#2 C:\xampp\htdocs\comple\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\comple\system\Common.php(1119): CodeIgniter\View\View->render('jefes/constanci...', Array, true)
#4 C:\xampp\htdocs\comple\app\Controllers\Jefes\ConstanciaController.php(43): view('jefes/constanci...', Array)
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Jefes\ConstanciaController->constancia()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Jefes\ConstanciaController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
>>>>>>> bda95cbc645214c86bb1c569ed45e25f9d6f44e5
